js/dataframe/Index.svelte

```diff
@@ -69,7 +68,9 @@
 				show_search: gradio.props.show_search,
 				pinned_columns: gradio.props.pinned_columns,
 				static_columns: gradio.props.static_columns,
-				fullscreen: gradio.props.fullscreen
+				fullscreen: gradio.props.fullscreen,
+				show_copy_button: gradio.props.buttons.includes("copy"),
```

Comment:
buttons can be undefined so this line causes failures in CI. Once that's fixed it should be good to go.

Reply:
thanks for the review @freddyaboulton!